earthquakes.py

```diff
@@ -31,31 +31,73 @@ def get_data():
 
     # We need to interpret the text to get values that we can work with.
     # What format is the text in? How can we load the values?
-    return ...
+    return json.loads(text)
 
 def count_earthquakes(data):
     """Get the total number of earthquakes in the response."""
-    return ...
+    return data.get("metadata").get("count")
 
 
 def get_magnitude(earthquake):
     """Retrive the magnitude of an earthquake item."""
-    return ...
+    return [dict.get("properties").get("mag") for dict in data.get("features")]
```

Comment:
You can make this shorter and (in my opinion) more readable by using `[...]` to access the dictionary contents:
```suggestion
    return [dict["properties"]["mag"] for dict in data["features"]
```
The `.get()` method is more useful when you want to retrieve a default value if the key is not found, rather than throw an error.

By the way, your function is doing something slightly different to what was intended: you are getting the magnitudes of all the earthquakes, rather than a single one. This is absolutely fine, but it doesn't match the documentation string exactly anymore :)